tests/testthat/test-occ_download.R

```diff
@@ -50,7 +50,7 @@ test_that("occ_download: real requests work", {
   expect_is(z, "occ_download")
   expect_is(unclass(z), "character")
   expect_match(unclass(z)[1], "[0-9]+-[0-9]+")
-  expect_equal(attr(z, "user"), "sckott")
-  expect_equal(attr(z, "email"), "myrmecocystus@gmail.com")
+  expect_equal(attr(z, "user"), "jwaller")
```

Comment:
Is there any reason why we cannot make this GBIF_USER environmental variable?

Reply:
Go for it. Sounds good